admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-choices-selector.ts

```diff
@@ -40,16 +41,43 @@ export default class BulkChoicesSelector {
 
   private tabContainer: HTMLDivElement;
 
-  constructor(eventEmitter: EventEmitter, tabContainer: HTMLDivElement) {
+  private paginatedCombinationsService: PaginatedCombinationsService
+
+  constructor(
+    eventEmitter: EventEmitter,
+    tabContainer: HTMLDivElement,
+    paginatedCombinationsService: PaginatedCombinationsService,
```

Comment:
You might need to inject the paginator as well in order to get the total elements in the list

Reply:
Either that or we need an even to be aware of the pagination updates (well mostly for the total)